packages/website/src/bin/page-generator/templates/example/test.js

```diff
@@ -1,10 +1,21 @@
 import assertValidTemplate from '../test-utils';
-import exampleTemplate from './index';
+import { exampleTemplate, exampleWithDecoratorTemplate } from './index';
 
 describe('example page template', () => {
-  it('creates valid source code for a page', () => {
+  it('creates valid source code for an example page', () => {
     const source = exampleTemplate('./component/path', './wrapper/path');
 
     assertValidTemplate(source);
   });
+
+  it('creates valid source code for an example with decorator page', () => {
+    const source = exampleWithDecoratorTemplate(
+      './component/path',
+      './wrapper/path',
+      {},
+      './decorator/path',
+    );
+
+    assertValidTemplate(source);
```

Comment:
I think it might also be worth asserting that the children of the wrapper contain the decorator rather than just checking that the file imports React. e.g. `expect(source).toMatch(/<Decorator>.*<Component \/>.*<\/Decorator>/s)`